### ISSUE TITLE
add labels for accessibility to buttons represented graphically by an icon

### ISSUE THREAD

**Issue by devil418** [notable/notable#632]
- **OS Version**: not relevant
- **Notable Version**: 1.4.0

### Current behavior

All buttons that use icons (i.e. paperclip, encil etc). don't have meaningful labels for screen readers to use. Screen readers, programs that read what's on the screen for visually impaired people, cannot deal properly with such buttons. Fortunately, the icon designers were considerate enough to give them labels, so there is at least some possibility of figuring out what the buttons do. For example, the edit button is read as "pencil", not "edit". A similar situation appears in all (or most) places where icons are used.

### Expected behavior
All buttons should have meaningful labels, i.e. the pencil button should be read as "edit", not pencil. The icons should be hidden entirely. In a few places, adding any labels isn't even necessary, it would be enough to hide the buttons. For example, the "notebooks" section in the sidebar already has the notebooks label, and the notebook icon does not have to be read at all. The suggested approach is to make some kind of component for providing accessible buttons. The component should hide the underlying icon with setting the aria-hidden attribute **of that icon, not of the button** to true. It should also provide an alternative, accessible description, passed via required props, so that no developer ever forgets about including one again. The description should be visible to screen readers only, as explained [here](https://stackoverflow.com/questions/26032089/in-html-how-can-i-have-text-that-is-only-accessible-for-screen-readers-i-e-fo)
